src/langgraph-platform/custom-middleware.mdx

```diff
@@ -58,6 +58,26 @@ Add the following to your `langgraph.json` configuration file. Make sure the pat
 }
 ```
 
+### Customize middleware ordering
+By default, custom middleware runs before authentication and authorization logic. To run custom middleware _after_ authentication and authorization, set `middleware_order` to `auth_first` in your `http` configuration.
```

Comment:
Maybe say min server version for this?

Reply:
Maybe:

> This customization is supported starting with API server v0.4.35 and later.

Reply:
ah also 'authorization' still runs after middleware sorry